### PR TITLE
Feat/panic

### PR DESCRIPTION
**Pull request by xl3ehindTim**

### CODE REVIEW

File: src/service/generateImageService.ts [hunk 1]
```diff
@@ -20,26 +20,31 @@ export const generateImages = async (): Promise<void> => {
...
-        `Sending request towards Stable Diffusion API for prompt ${json.prompt}, ${i}`
-      );
-      const response = await axios.post(
-        `${apiUrl}/sdapi/v1/${endpoint}2img`,
-        json
-      );
+    try {
+      for (let i = 0; i < 10; i++) {
+        // Filter the prompt before handing it in to the "getJson" method
+        const { json, endpoint } = await getJson(prompt);
```
Comment (BartvanEijkelenburg) [inline src/service/generateImageService.ts:27]: What if we use the panic button on the panic?

Comment (xl3ehindTim) [reply src/service/generateImageService.ts:27]: It will crash, and continue. Therefore I added the try/except as a simple solution

Comment (BartvanEijkelenburg) [reply src/service/generateImageService.ts:27]: Will it be able to fix the panic issue? I am not able to test it currently but will soon.

Comment (xl3ehindTim) [reply src/service/generateImageService.ts:27]: What issue do you mean?

Comment (BartvanEijkelenburg) [reply src/service/generateImageService.ts:27]: If we panic there is a reason to panic, will this solution be sufficient to remove any explicit content, so what if we use 2 panics to get rid of it, not ideal ofcourse.

But I will test this asap

**Pull request merged**